moshi/japicmp/build.gradle.kts

```diff
@@ -33,7 +33,8 @@ val japicmp = tasks.register<JapicmpTask>("japicmp") {
     "com.squareup.moshi.RecordJsonAdapter\$ComponentBinding", // Package-private
   )
   methodExcludes = listOf(
-    "com.squareup.moshi.JsonAdapter#indent(java.lang.String)" // Was unintentionally open before
+    "com.squareup.moshi.JsonAdapter#indent(java.lang.String)", // Was unintentionally open before
+    "com.squareup.moshi.AdapterMethodsFactory#get(java.lang.Object)" // Internal.
```

Comment:
Just exclude the whole class as it's package-private

Reply:
https://github.com/square/moshi/pull/1467/commits/5e9c4c3de15ad3d796db3d52bdc88a3a8396b58a